src/pages/sourceSettings/sourceSettings.tsx

```diff
@@ -121,40 +120,21 @@ class SourceSettings extends React.Component<Props, State> {
     const columns = [
       t('source_details.column.name'),
       t('source_details.column.type'),
-      t('source_details.column.last_contacted'),
+      t('source_details.column.added_by'),
+      t('source_details.column.date_added'),
```

Comment:
Instead of providing `t` as a property, it would be preferred to wrap classes with `translate`. We do this in several classes like LoadingState, OcpReportSummary, and even MessageDialogBase, etc. 

We don't necessarily need to connect; for example, it can be as simple as:

`const OcpReportSummary = translate()(OcpReportSummaryBase);`